packages/react/src/auto/polaris/inputs/PolarisDateTimePicker.tsx

```diff
@@ -0,0 +1,117 @@
+export const PolarisDateTimePicker = (props: {
+  field: string;
+  id?: string;
+  value?: Date;
+  timeLabel?: string;
+  onChange?: (value: Date) => void;
+  error?: string;
+  includeTime?: boolean;
+  hideTimePopover?: boolean;
+  datePickerProps?: Partial<DatePickerProps>;
+  timePickerProps?: Partial<TextFieldProps>;
+}) => {
+  const { path, metadata } = useFieldMetadata(props.field);
+
+  const { field: fieldProps } = useController({
+    name: path,
+  });
+
+  const { onChange, value } = props;
+  const localTz = Intl.DateTimeFormat().resolvedOptions().timeZone;
+  const localTime = value ? utcToZonedTime(value, localTz) : fieldProps.value;
+  const [dateTimeState, setDateTimeState] = useState({
+    month: (localTime ?? today).getMonth(),
+    year: (localTime ?? today).getFullYear(),
+    day: (localTime ?? today).getDay(),
+    hour: (localTime ?? today).getHours().toString(),
+    minute: (localTime ?? today).getMinutes().toString().padStart(2, "0"),
+    ampm: (localTime ?? today).getHours() >= 12 ? "PM" : "AM",
+  });
+  const [datePopoverActive, setDatePopoverActive] = useState(false);
+
+  const config = metadata.configuration;
+
+  const onDateChange = useCallback<Exclude<DatePickerProps["onChange"], undefined>>(
+    (range) => {
+      const date = new Date(range.start);
+      localTime && copyTime(date, localTime);
+      onChange?.(zonedTimeToUtc(date, localTz));
+      fieldProps.onChange(zonedTimeToUtc(date, localTz));
+      setDatePopoverActive(false);
+    },
+    [localTime, onChange, localTz, fieldProps]
+  );
+
+  const toggleDatePopoverActive = useCallback(() => setDatePopoverActive((active) => !active), []);
+  const handleMonthChange = useCallback((month: number, year: number) => {
+    setDateTimeState((prevState) => ({
+      ...prevState,
+      month,
+      year,
+    }));
+    // setDateTimeState({ ...dateTimeState, month, year });
+  }, []);
+
+  return (
+    <InlineStack gap="400">
+      <Popover
+        preferredPosition="above"
+        active={datePopoverActive}
+        activator={
+          <TextField
+            id={props.id ? `${props.id}-date` : undefined}
+            label={metadata.name ?? "Date"}
+            prefix={<Icon source={CalendarIcon} />}
+            autoComplete="off"
+            value={localTime ? format(localTime, "yyyy-MM-dd") : ""}
+            onFocus={toggleDatePopoverActive}
+            error={props.error}
+          />
+        }
+        onClose={toggleDatePopoverActive}
+      >
+        <div style={{ padding: "16px" }}>
+          <DatePicker
+            month={dateTimeState.month}
+            year={dateTimeState.year}
+            allowRange={false}
+            onChange={onDateChange}
+            onMonthChange={handleMonthChange}
+            selected={props.value}
+            {...props.datePickerProps}
+          />
+        </div>
+      </Popover>
+      {(props.includeTime ?? (config as any).includeTime) && (
+        <PolarisTimePicker
```

Comment:
add `{...props.timePickerProps}`

Reply:
I added `{...props.timePickerProps}` to the `TextField` activator in `PolarisTimePicker`